packages/cxl-ui/scss/cxl-course-card.scss

```diff
@@ -1,10 +1,11 @@
 /* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix -- some of these are necessary for line-clamp implementation */
-/* stylelint-disable declaration-no-important  */
+/* stylelint-disable declaration-no-important, order/properties-order */
```

Comment:
why properties-order rule is bad and you removed it?

Reply:
It's too much of a headache remembering the CSS Spec. I prefer alphabetical.

Reply:
Isn't it corrected automatically by linter?

Reply:
It's not :/ I don't mind using the default rules, though we could also use this? https://github.com/hudochenkov/stylelint-order/tree/master/rules/properties-alphabetical-order